pkg/web/apps/apps_http_handler_test.go

```diff
@@ -51,21 +56,25 @@ func Test_HttpHandler_GetAppsById(t *testing.T) {
 	c := e.NewContext(req, rec)
 	c.SetPath("/api/apps/:id")
 	c.SetParamNames("id")
-	c.SetParamValues(helpers.GetMockApp().AppID)
-	h := &httpHandler{mockedAppService}
+	c.SetParamValues(app.ID)
+	h := NewHTTPHandler(e, mockedAppService)
 
 	// Assertions
 	if assert.NoError(t, h.GetAppByID(c)) {
 		assert.Equal(t, http.StatusOK, rec.Code)
-		//TODO: It will be fixed when the handler be implemented
-		// appJson, _ := json.Marshal(helpers.GetMockApp())
-		//assert.Equal(t, string(appJson)+"\n", rec.Body.String())
+		appJSON, _ := json.Marshal(helpers.GetMockApp())
+		assert.Equal(t, string(appJSON)+"\n", rec.Body.String())
 	}
 }
 
+//TODO when update app endpoint created its hitting get apps by id endpoint at the moment
 func Test_HttpHandler_UpdateApp(t *testing.T) {
 	// make and configure a mocked Service
-	mockedAppService := &AppServiceMock{
+	app := helpers.GetMockApp()
+	mockedAppService := &ServiceMock{
+		GetAppByIDFunc: func(ID string) (*models.App, error) {
+			return app, nil
+		},
 		GetAppsFunc: func() (*[]models.App, error) {
```

Comment:
You can remove the GetAppSFun since it is not used in this test.

Reply:
rewrote this test